composer.json

```diff
@@ -114,6 +139,14 @@
         "sendgrid/sendgrid": "~6.0"
     },
     "require-dev": {
+
```

Comment:
None of these are really dependencies for development. For example, if someone doesn't have xdebug installed, would you stop them from installing mautic w/ dev dependencies? Surely not. 

I'd drop all of these personally as it's not worth the headache.